### PR TITLE
Volume FT fixes

### PR DESCRIPTION
**Pull request by doriac11**
# Description
- A few minor fixes after writing the volumes Functional Tests. 

# ISSUE TYPE
- Bugfix Pull Request

##### RESOURCE OR DATASOURCE NAME
Volume
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

**Comment by doriac11** [comment]
run e2e test

**Comment by doriac11** [comment]
run e2e test

File: powermax/provider/volume_resource_test.go [hunk 1]
```diff
@@ -152,7 +152,12 @@ func TestAccVolumeResource_Error_Updating(t *testing.T) {
 			// Invalid name
 			{
 				Config:      ProviderConfig + VolumeConfigInvalidName,
-				ExpectError: regexp.MustCompile("Failed to rename volume"),
+				ExpectError: regexp.MustCompile("Invalid Attribute Value Match"),
+			},
+			// Normal Config
```
Comment (doriac11) [inline powermax/provider/volume_resource_test.go:157]: To make sure the resource gets cleaned up correctly

File: powermax/provider/volume_resource.go [hunk 2]
```diff
@@ -77,6 +78,11 @@ func (r volumeResource) Schema(ctx context.Context, req resource.SchemaRequest,
 				Required:            true,
 				Validators: []validator.String{
 					stringvalidator.LengthAtLeast(1),
+					stringvalidator.LengthAtMost(64),
```
Comment (shekhar-j) [inline powermax/provider/volume_resource.go:81]: @doriac11 can you update description also like we did for all others?

**Review by shekhar-j — APPROVED** [review]

File: powermax/provider/volume_resource.go (hunk 2 above)
Comment (doriac11) [reply powermax/provider/volume_resource.go:81]: Ahh yes I can

**Comment by doriac11** [comment]
run e2e tests

File: powermax/provider/volume_resource.go (hunk 2 above)
Comment (doriac11) [reply powermax/provider/volume_resource.go:81]: Done

**Review by nikitajoshi1 — APPROVED** [review]

**Comment by doriac11** [comment]
run e2e test

**Review by shekhar-j — APPROVED** [review]

**Pull request merged**